tailwind.config.js

```diff
@@ -57,6 +57,11 @@ module.exports = {
         ':merge(.group):focus-within &',
         ':merge(.group):hover &',
       ]);
+      /**
+       * Note: in TW v3.4.0, the *: variant is added for targeting direct children
+       * https://github.com/tailwindlabs/tailwindcss/pull/12551
+       * Leaving these in for now for backwards compatibility.
+       */
```

Comment:
Leaving in our current custom children: variant because we used that on every site.

Reply:
Yeah, and they are different. 

Children use the direct child selector to select any immediate element `> *`  which scopes that at one level
I'd have to test to confirm but using the wildcard selector `*` without the direct child selector opens up the selectivity scope to all the levels

Reply:
Oh don't worry @sherakama , Tailwind actually make the *: variant only select direct childrens so > *, same as our children: variant. Some people were commenting on whether it's good idea for them to pick the * 😂 

![Screenshot 2024-01-11 at 10 01 50 AM](https://github.com/SU-SWS/decanter/assets/42749717/3a6a1cf0-bf1b-4c35-9b48-81e81859da75)

Reply:
Oh, I just read https://github.com/tailwindlabs/tailwindcss/pull/12551

`*` is a direct child selector 🤦 

That difference isn't obvious with this syntax but it makes sense after reading the thread.

Reply:
Yeah it could be a bit confusing because of the * 😅